ansible/roles/ansible-tvault-contego-extension/tasks/set_virtenv.yml

```diff
@@ -103,7 +103,7 @@
     - name: activate the virtual environment and set cryptography symlinks when openstack_release is newton
       shell: '{{virtenv_premitaka_script_path}}'
```

Comment:
Does this shell script handles the queens release version ?

Reply:
yes.. that shell script install tvault-contego pip pacakge in queens virtual environment